### ISSUE TITLE
Enhanced Steam loads all 62fx62f thumbnail images of my inventory on market page

### ISSUE THREAD

**Issue by ryanli** [jshackles/Enhanced_Steam#922]
This is extremely slow for users with large inventories — in my case I have 1000+ items and the page is unresponsive for almost half a minute due to loading all the assets.

**Comment by JoiNNN** [comment]
You mean on [http://steamcommunity.com/market/](http://steamcommunity.com/market/)? The only items I see loading with that thumbnail are the ones under **Popular Items, Newly Listed, Recently Sold** and not the actual listings.

**Comment by ryanli** [comment]
Yes, I can see the thumbnails being loaded in Chrome's network console although I have no idea whether/how the thumbnails are used.

**Comment by JoiNNN** [comment]
This should be fixed with the latest market changes Valve made.
Can be closed.

**Comment by VanityFox** [comment]
So, Enhanced Steam is discontinued. There is now [Augmented Steam](https://github.com/tfedor/AugmentedSteam/). Perhaps your requested feature/bugfix is already implemented there! If not, you could move your issue there.
